internal/provider/models/deployment.go

```diff
@@ -459,11 +441,11 @@ func WorkerQueueDataSourceTypesObject(
 }
 
 type DeploymentScalingSpec struct {
-	HibernationSpec HibernationSpec `tfsdk:"hibernation_spec"`
```

Comment:
had to change it to use types.Object for resource creation since it would error otherwise